package.json

```diff
@@ -4,6 +4,11 @@
   "description": "An ESLint config for JavaScript projects, brought to you by Netflix",
   "type": "module",
   "main": "index.js",
+  "module": "index.js",
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org"
+  },
```

Comment:
Just making a couple updates as we work to get out GitHub Actions to publish ironed out. Part of #22.